sycl/include/sycl/accessor.hpp

```diff
@@ -833,7 +834,7 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
             /*AccessMode=*/getAdjustedMode({}),
             /*SYCLMemObject=*/nullptr, /*Dims=*/0, /*ElemSize=*/0,
             /*IsPlaceH=*/false,
-            /*OffsetInBytes=*/0, /*IsSubBuffer=*/false, /*PropertyList=*/{}){};
+            /*OffsetInBytes=*/0, /*IsSubBuffer=*/false, /*PropertyList=*/{}) {};
```

Comment:
I see a lot of unrelated, formatting-related changes. Given that this PR modifies several files, can we separate out these unrelated changes into a separate NFC PR? That would be make it easier for me to review the changes.

Reply:
`clang-format` did indeed suggest this change.

> I see a lot of unrelated, formatting-related changes. Given that this PR modifies several files, can we separate out these unrelated changes into a separate NFC PR? That would make it easier for me to review the changes.

There is definitely some formatting changes, but most of the formatting changes are coupled with real changes, e.g. `const decltype(Obj::impl) &getSyclObjImpl(...` instead of `decltype(Obj::impl) getSyclObjImpl(...`. I can separate these into another PR, but CI would break on both PR's as the code would be invalid. Could I perhaps point out the changes that matter, or do you still prefer a separate PR even though CI would break?